pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/KeyValueSchema.java

```diff
@@ -22,6 +22,7 @@
 
 import java.nio.ByteBuffer;
 
+import com.google.gson.Gson;
```

Comment:
Can this extra dependency be removed? The client already shades a json library from pulsar-common and we should use that here and try to avoid extra dependencies.

Reply:
@cckellogg what does it matter here?

Gson is already a dependency in pulsar-client. It is not a new dependency. It is already used in AuthenticationBasic.java

Reply:
DIdn't realize that there are so many shaded things I guess I missed it. I guess it doesn't matter.

Reply:
@cckellogg we don't need to use the shaded dependencies here. we use the normal dependencies. the gson dependency will be shaded when we build `pulsar-client` jar.

Reply:
ok sounds good